css/properties/width.json

```
@@ -286,6 +286,54 @@
             }
           }
         },
+        "fit-content-function": {
```

I don't think we actually document this anywhere, but we typically use underscores for word spaces and dashes literally.

```suggestion
        "fit-content_function": {
```